db/migrate/20220620084648_add_email_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailToUsers < ActiveRecord::Migration[6.0]
```

Comment:
We had `email` column in `users` table. If you want to add not-null condition, you can create another migration script.

Reply:
I see, I will use additional migrations instead of altering existing migrations in the future.